cypress/plugins/index.js

```diff
@@ -1,39 +1,18 @@
 const webpack = require('@cypress/webpack-preprocessor');
 const { initPlugin: initSnapshotPlugin } = require('cypress-plugin-snapshots/plugin');
 
+const webpackFilename = __dirname + '/webpack.config.js';
```

Comment:
It's not clear to me why this was needed in the first place but you probably want to use `path.join(__dirname, 'webpack.config.js')` instead to make Windows users happy.